cmd/otelcontribcol/go.mod

```diff
@@ -4,7 +4,7 @@ module github.com/open-telemetry/opentelemetry-collector-contrib/cmd/otelcontrib
 
 go 1.22.0
 
-toolchain go1.22.7
+toolchain go1.23.1
```

Comment:
> I don't think otel people would allow updating Go version in a PR that adds a new exporter. A PR should do 1 thing. I suggest to keep the Go version same.

Yeah this file has been deleted already